test/env.js

```diff
@@ -1 +1,5 @@
 process.env.METAMASK_ENV = 'test';
+
+process.env.ICON_NAMES = {
+  LOADING_FILLED: 'loading-filled',
+};
```

Comment:
Could we just add a comment here to explain what these environment variables are for?

```suggestion
/**
 * Used for testing components that use the Icon component 
 * 'ui/components/component-library/icon/icon.js'
 */
process.env.ICON_NAMES = {
  LOADING_FILLED: 'loading-filled',
};
```

@Gudahtt This should probably be automated so the icon name environment variables and testing variables don't get out of sync right? I think it's outside the scope of this PR but I have another PR open that could address this
https://github.com/MetaMask/metamask-extension/issues/16046